mobile/ios/Runner/Info.plist

```diff
@@ -86,11 +86,23 @@
       <dict>
         <key>CFBundleTypeRole</key>
         <string>Editor</string>
+        <key>CFBundleURLName</key>
+        <string>Share Extension</string>
```

Comment:
Why is this added?

Reply:
I gave titles to the URLs since there are multiple in the app now. Doesn't affect functionality, just gives a name